deploy/crds/ocs_v1alpha1_storagecluster_cr.yaml

```diff
@@ -5,3 +5,17 @@ metadata:
   name: example-storagecluster
 spec:
   manageNodes: false
+  storageDeviceSets:
+  - name: example-deviceset
+    count: 3
+    resources: {}
+    placement: {}
+    dataPVCTemplate:
+      spec:
+        storageClassName: gp2
```

Comment:
OK. I see... this is where the `gp2` name has its origin. Somewhat weird to have it defined in an example (is it?), and referenced hard in the code..